src/6_branch.js

```diff
@@ -1009,7 +1009,7 @@ Branch.prototype['sendSMS'] = wrap(
 					self._api(
 						resources.linkClick,
 						{
-							"link_url": 'l/' + url.split('/').pop(),
+							"link_url": url.substring(url.lastIndexOf(url.match(/\/\w\//)[0])),
```

Comment:
not sure if this does this - but we need to get two url fragments - so we want to support the following usecases:

- `http://bnc.lt/l/abcdef` -> `l/abcdef`
- `http://abc.com/l/abcdef` -> `l/abcdef`
- `http://bnc.lt/m/abcdef` -> `m/abcdef`
- `http://abc.com/m/abcdef` -> `m/abcdef`
- `http://bnc.lt/XYZ/abcdef` -> `XYZ/abcdef`
- `http://abc.com/XYZ/abcdef` -> `XYZ/abcdef`